pkg/go.mod

```diff
@@ -46,7 +46,7 @@ require (
 	github.com/xeipuuv/gojsonschema v1.2.0
 	github.com/zclconf/go-cty v1.13.2
 	gocloud.dev v0.37.0
-	gocloud.dev/secrets/hashivault v0.27.0
+	gocloud.dev/secrets/hashivault v0.37.0
```

Comment:
This was the change I made. All the other changes were the result of running `make tidy`.